src/redux/categories/Categories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const STATUS = 'STATUS';
+
+export function createWidget(category) {
+  return { type: STATUS, category };
+}
+
+const initialState = {
+  categories: [],
+};
+
+export default function categories(state = initialState, action) {
+  switch (action.type) {
+    case STATUS:
+      return { ...state, categories: action.categories };
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] Case action.type is `CHECK_STATUS` the function should return a string _"Under construction"_. Please improve upon this as per this module's requirement